src/chain.h

```diff
@@ -14,6 +14,20 @@
 
 #include <vector>
 
+/**
+ * Maximum amount of time that a block timestamp is allowed to exceed the
+ * current network-adjusted time before the block will be accepted.
+ */
+static const int64_t MAX_FUTURE_BLOCK_TIME = 2 * 60 * 60;
```

Comment:
> This one belongs in consensus/consensus.h

I don't have an opinion, but when I discussed this with @morcos a while ago, I think he suggested it shouldn't go there because it isn't a constant that needs to be exactly the same everywhere to have consensus.